### ISSUE TITLE
Cancelling an "edit" in a ToMany morphic list -> nil

### ISSUE THREAD

**Issue by GoogleCodeExporter** [magritte-metamodel/magritte#17]
```
In Morphic:
1. Click the "Edit" button on an item in a ToMany list
2. In the form that pops up, click cancel

The item label is 'nil', but if you open the container freshly, the item is 
still there intact.
```

Original issue reported on code.google.com by `sean.p.d...@gmail.com` on 26 Sep 2013 at 3:52

**Comment by noha** [comment]
Is this still the case?

**Comment by seandenigris** [comment]
Seems to be fixed, possibly by #121
